plugin/rewrite/README.md

```diff
@@ -96,3 +96,24 @@ rewrite edns0 subnet set 24 56
 
 * If the query has source IP as IPv4, the first 24 bits in the IP will be the network subnet.
 * If the query has source IP as IPv6, the first 56 bits in the IP will be the network subnet.
+
+### Substrings and Regular Expressions
+
+The `sub:` substring in `FROM` part of the `rewrite` syntax allows to match based
```

Comment:
Wouldn't it be better to do this in the `FIELD` part, like `name_sub` or `name_regexp` ?

Reply:
@miekg , my thinking is that `name` is the `FIELD` and I would think that leaving it intact makes more sense because `name_sub` and `name_regexp` sound more line an "action".

In my mind having `sub:` and `regex:` as "prefix" works best in `FROM`, because, in theory, there should be no "name" with `:` (colon) in it.

However, as someone who coded my favorite DNS library 🥇  , I will side with whatever choice you prefer 😄